src/app/index.tsx

```diff
@@ -1,21 +1,32 @@
 import 'react-native-gesture-handler';
 import React from 'react';
+import { Image } from 'react-native';
 import { NavigationContainer } from '@react-navigation/native';
 import { createStackNavigator } from '@react-navigation/stack';
 import { RootStackParamList } from '@interfaces/navigation';
 import Routes from '@constants/routes';
+import goBackIcon from '@assets/ic_back.png';
+import headerBackground from '@assets/bc_navbar.png';
 
-import BookList from './screens/BookList';
 import BookDetail from './screens/BookDetail';
+import BookList from './screens/BookList';
+import styles from './styles';
 
 const { Navigator, Screen } = createStackNavigator<RootStackParamList>();
 
 function App() {
+  const stackNavigatorConfig = {
```

Comment:
Let's move this to `src/config/navigation.tsx`. You can put the styles in that same file. Like 
```
stackNavigatorConfig:{
 headerTitleStyle: {
     fontSize: 16,
     color: white,
     fontWeight: '600',
     letterSpacing: -0.4
 },
...
```

Reply:
I like that